source/reference/method/MongoDBDatabase-aggregate.txt

```diff
@@ -54,11 +54,13 @@ Parameters
 
      * - batchSize
        - integer
-       - Specifies the batch size for the cursor, which will apply to both the
-         initial ``aggregate`` command and any subsequent ``getMore`` commands.
-         This determines the maximum number of documents to return in each
-         response from the server.
```

Comment:
I think we can discuss `batchSize: 0` here as we do for the `find()` option, since there is a legitimate use case for doing so.

There is a special directive in the [CRUD spec](https://github.com/mongodb/specifications/blob/master/source/crud/crud.md) for drivers to intentionally omit `batchSize` for pipelines that include `$out` or `$merge`, since it might prevent the pipeline from executing (and writing output). I don't think we need to mention that, though, as it is enforced internally (see: [Aggregate.php](https://github.com/mongodb/mongo-php-library/blob/1.21.1/src/Operation/Aggregate.php#L204-L209)).